tests/test_expt_file_counts_handler.py

```diff
@@ -0,0 +1,67 @@
+"""
+Copyright 2023 NOAA
+All rights reserved.
+
+Unit tests for expt_file_counts.py
+
+"""
+import os
+import pathlib
+import pytest
+import json
+
+
+from expt_file_counts import ExptFileCountRequest
+
+def test_put_expt_file_counts_request_dict():
+    request_dict = {
+        'name': 'expt_file_counts',
+        'method': 'PUT',
+        'body': {
+            'experiment_name': 'C96L64.UFSRNR.GSI_3DVAR.012016',
+            'wallclock_start': '2021-07-22 09:22:05',
+            'file_type_name': 'example_type',
+            'file_extension': '.example',
+            'time_valid' : '2023-02-03 04:54:23',
```

Comment:
note that time_valid here is smaller than the cycle value. This seem strange to me.

Reply:
I understood time valid as the time in which the value referred to was accurate so the current time the metric was calculated in case those files change in a new run of an experiment. I was following the pattern also used for experiment metrics. How did you understand time valid being used? I think we should have a further discussion about the table schema and column usage before we finalize it for use in production

Reply:
Let chat about this one in person on tuesday.

Reply:
updated based on in person discussion